FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -73,20 +73,25 @@ final class ComposeMessageService {
         self.onStateChanged = completion
     }
 
-    func prepareSigningKey(viewController: UIViewController?) async throws -> PrvKeyInfo? {
-        guard let viewController = viewController, let signingKey = try await keyService.getSigningKey(email: sender) else {
+    func prepareSigningKey(viewController: UIViewController?) async throws -> Keypair? {
+        // todo - sender email will differ from account email once
+        //   https://github.com/FlowCrypt/flowcrypt-ios/issues/1298 is done
+        let keys = try await keyAndPassPhraseStorage.getKeypairsWithPassPhrases(email: sender)
+        guard var signingKey = try await keyMethods.chooseSenderSigningKey(keys: keys, senderEmail: sender) else {
+            // todo - throw user error for missing signing key
             return nil
         }
```

Comment:
potentially todo in this PR. If not, then I'll file an issue or link existing issue